eqc/tcp_proto_eqc.erl

```diff
@@ -83,6 +83,12 @@ resolution() ->
 ttl() ->
     oneof([infinity, mtime()]).
 
+read_repair_opt() ->
+    oneof([{rr, default}, {rr, on}, {rr, default}]).
```

Comment:
This lists 'default' twice and no off.

also `{rr, oneof([default, on, off])}` will read more stright forward (and would probably have made the missing off more obvious ;).